### DISCUSSION TITLE
20GB are to small. How i can change it to 32/ 64 GB?

### DISCUSSION THREAD

**Discussion by iCRON862** [utmapp/UTM#4743] [UTM Help (macOS)]
**Describe the issue**
If i use a prebuild of Windows 7 in UTM the hard disk size are 20 GB witch are to small to install all the updates from windows 7. How can i change/ upgrade it to 32/ 64 GB?

**Configuration (required)**
* UTM Version: 4.0.9
* OS Version: iPadOS 14
* Device Model: iPad Pro 2020
* Is it jailbroken (name jailbreak used)? Non-Jailbreak
* How did you install UTM? TrollStore

**Crash log**
If the app crashed, you need a crash log. To get your crash log, open the Settings app and browse to `Privacy -> Analytics & Improvements -> Analytics Data` and find the latest entry for UTM. You should export the text and attach it here.

N/A

**Debug log**
For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the bottom of the configuration options (or at the top of the `QEMU` page) is `Debug Log`. Turn it on and save the VM. After you experience the issue, quit the VM and re-launch UTM. Open the VM settings again and select `Export Log...` and attach it here.

N/A

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since GitHub has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by osy** [comment]
It’s not supported on iOS but you can move it to a Mac, resize it, and move it back to iOS.

**Reply by iCRON862** [reply]
Ok, i have tryed without a prebuild. But there i got errors that autounattend.xml was corrupted/ wrong or the cd/dvd device drivers are missing. I don't know how i add a working autounattend.xml or the cd/dvd drivers into the vm. Google didn't helped me out.

**Reply by iCRON862** [reply]
> It’s not supported on iOS but you can move it to a Mac, resize it, and move it back to iOS.

No more needed. I have fixed the issue by enable  'Install Windows 10 or highter' and instead of windows 10 i used my Windows 7 iso. Then it was possible to change the size for the vm. Booted and installed Win7 successfully. Thx
